scanner/src/main/kotlin/scanners/scanoss/ScanOssResultParser.kt

```diff
@@ -32,18 +32,6 @@ import org.ossreviewtoolkit.utils.spdx.SpdxConstants
 import org.ossreviewtoolkit.utils.spdx.SpdxExpression
 import org.ossreviewtoolkit.utils.spdx.calculatePackageVerificationCode
 
-/**
```

Comment:
Please keep this. The whole file is not used yet, as it was a preparation for adding SCANOSS as a scanner.